### PR TITLE
3: ADD A SERVLET

### PR DESCRIPTION
**Pull request by TWilson2017**
This code is the RegisterServlet class, RegisterServletTest class, and a modified web.xml to showcase:
image

### CODE REVIEW

**Review by sherryx8 — APPROVED** [review]
everything looks good!

**Review by tharminhtet — APPROVED** [review]
Everything looks good, exactly as in the instruction.

File: src/main/java/codeu/controller/RegisterServlet.java [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+* Servlet class responsible for user registration.
+*/
+public class RegisterServlet extends HttpServlet {
+
+ @Override
+ public void doGet(HttpServletRequest request, HttpServletResponse response)
+     throws IOException, ServletException {
+
+   response.getWriter().println("<h1>RegisterServlet GET request.</h1>");
+ }//doGet
```
Comment (TsueiCat) [inline src/main/java/codeu/controller/RegisterServlet.java]: This is a neat way of marking the ends of methods and classes!

**Review by TsueiCat — APPROVED** [review]
Nice work!

File: src/main/java/codeu/controller/RegisterServlet.java [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/**
+* Servlet class responsible for user registration.
+*/
+public class RegisterServlet extends HttpServlet {
+
+ @Override
```
Comment (sgoya) [inline src/main/java/codeu/controller/RegisterServlet.java]: I think the spacing in your java files is a little inconsistent. We want to have the same indent everywhere. So if our indent is 2 spaces, we want to do it for all indents. I think you have a 1 space indent for you class methods/attributes. Please fix in both java files.

**Pull request merged**